dhash/database.py

```diff
@@ -102,6 +103,59 @@ def dump(self) -> dict:
         }
 
 
+class HashConfig(database.base):
+    """Stores config in format key:value"""
+
+    __tablename__ = "fun_dhash_config"
+
+    key = Column(String, primary_key=True)
+    value = Column(String)
+
+    @staticmethod
+    def init(key: str, value) -> str:
+        config = session.query(HashConfig).filter_by(key=key).one_or_none()
+
+        if config:
+            return False
+
+        config = HashConfig(key=key, value=value)
+
+        session.add(config)
+        session.commit()
+
+        return True
+
+    @staticmethod
+    def get(key: str, value=None) -> str:
+        config = session.query(HashConfig).filter_by(key=key).one_or_none()
+
+        if config:
+            return config.value
+        else:
+            return value
+
+    @staticmethod
+    def set(key: str, value: str):
+        config = session.query(HashConfig).filter_by(key=key).one_or_none()
+
+        if not config:
+            config = HashConfig(key=key)
+
+        config.value = value
+
+        session.merge(config)
+        session.commit()
+
+    def __repr__(self) -> str:
+        return f'<{self.__class__.__name} key="{self.key}" value="{self.value}">'
```

Comment:
*`__name__`

Reply:
Oh, I've stolen that from ImageHash. Fixed both.